webapp/components/experiments/Experiments.js

```diff
@@ -32,17 +52,89 @@ const Experiments = (props) => {
     props.viewExperiment(payload);
   };
 
+  const handleDelete = () => {
+    console.info('You clicked the delete icon.');
+  };
+
+  const handleClick = () => {
+    console.info('You clicked the Chip.');
+  };
+
+  const formatDate = (timestamp) => {
+    const date = new Date(timestamp)
+    return date?.toLocaleDateString().replaceAll('/', '.');
```

Comment:
Would remove the str replace. While the design shows the European format we want to display the date format of the user's configured locale

Reply:
done @lrebscher